learn/getting_started/quick_start.md

```diff
@@ -10,6 +10,16 @@ Let's get started!
 
 We'll start with downloading and installing Meilisearch. You have the option to install Meilisearch locally or deploy it over a cloud service.
 
+### Supported operating systems
+
+The following operating systems are compatible with the Meilisearch binary:
```

Comment:
Do users gain much from us specifying  compatibility with the binary? Can an OS be compatible with Meilisearch but incompatible with the binary?

```suggestion
The following operating systems are compatible with Meilisearch:
```